addons/auth/ember-cli-build.js

```diff
@@ -4,7 +4,10 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function (defaults) {
   let app = new EmberAddon(defaults, {
-    // Add options here
+    'ember-simple-auth': {
+      // https://github.com/mainmatter/ember-simple-auth/blob/master/guides/upgrade-to-v4.md#call-sessionsetup-method-in-your-applicationroute
```

Comment:
Don't we have to call `this.session.setup();` manually now in our application routes according to the docs to turn this feature on?

Reply:
Answered this on a comment below 😉  thank you for the feedback!